fs/xfstests.py

```diff
@@ -129,6 +134,59 @@ def setUp(self):
                 else:
                     self.cancel('Need %s GB to create loop devices' % check)
             self._create_loop_device(base_disk, loop_size, mount)
+        elif self.dev_type == 'nvdimm':
+            self.test_dev = self.params.get('disk_test', default=None)
```

Comment:
get this assignment as global and change value inside loop that seems me a good way

Reply:
As we have different way of handling each disk_type, i think it should not matter.